README.md

```diff
@@ -70,6 +70,22 @@ This requires Ruff version `v0.1.3` or later.
 
 [notebook document synchronization]: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#notebookDocument_synchronization
 
+## Untrusted Workspace
+
+_New in v2024.32.0_
```

Comment:
Period after the last 0? Or does this look right as-is?

Reply:
I think it looks fine as-is